packages/flutter_tools/bin/podhelper.rb

```diff
@@ -319,3 +324,36 @@ def flutter_relative_path_from_podfile(path)
   relative = pathname.relative_path_from project_directory_pathname
   relative.to_s
 end
+
+def flutter_parse_xcconfig_file(file)
+  file_abs_path = File.expand_path(file)
+  if !File.exists? file_abs_path
+    return [];
+  end
+  entries = Hash.new
+  skip_line_start_symbols = ["#", "/"]
+  File.foreach(file_abs_path) { |line|
+      next if skip_line_start_symbols.any? { |symbol| line =~ /^\s*#{symbol}/ }
+      key_value_pair = line.split(pattern='=')
+      if key_value_pair.length == 2
+        entries[key_value_pair[0].strip()] = key_value_pair[1].strip();
+      else
+        puts "Invalid key/value pair: #{line}"
+      end
```

Comment:
I think the whitespace is off here.
```suggestion
    next if skip_line_start_symbols.any? { |symbol| line =~ /^\s*#{symbol}/ }
    key_value_pair = line.split(pattern = '=')
    if key_value_pair.length == 2
      entries[key_value_pair[0].strip()] = key_value_pair[1].strip();
    else
      puts "Invalid key/value pair: #{line}"
    end
```